crux-http-server/src/crux/http_server/query.clj

```diff
@@ -3,35 +3,31 @@
             [clojure.java.io :as io]
             [clojure.spec.alpha :as s]
             [clojure.string :as string]
-            [cognitect.transit :as transit]
             [crux.api :as crux]
             [crux.codec :as c]
             [crux.error :as err]
             [crux.http-server.entity-ref :as entity-ref]
+            [crux.http-server.json :as http-json]
             [crux.http-server.util :as util]
             [crux.io :as cio]
-            [crux.query :as q]
             [muuntaja.core :as m]
             [muuntaja.format.core :as mfc]
             [muuntaja.format.edn :as mfe]
             [muuntaja.format.transit :as mft]
             [spec-tools.core :as st])
-  (:import crux.http_server.entity_ref.EntityRef
-           crux.io.Cursor
-           [java.io Closeable]
-           (java.io OutputStream Writer)
-           [java.time Instant ZonedDateTime ZoneId]
+  (:import [java.io Closeable OutputStream]
+           [java.time Instant ZoneId]
            java.time.format.DateTimeFormatter
            java.util.Date))
 
-(s/def ::query
+(s/def ::query-edn
   (st/spec
    {:spec any? ; checked by crux.query
     :decode/string (fn [_ q] (util/try-decode-edn q))}))
 
-;; TODO: Need to ensure all query clasues are present + coerced properly
+;; TODO: Need to ensure all query clauses are present + coerced properly
```

Comment:
Likely a redundant TODO - all clauses would be present + coerced when passed into `query-edn`